update.go

```diff
@@ -174,6 +194,18 @@ func (b *UpdateStmt) SetMap(m map[string]interface{}) *UpdateStmt {
 	return b
 }
 
+// Incr increases column by value
+func (b *UpdateStmt) IncrBy(column string, value interface{}) *UpdateStmt {
+	b.Value[column] = opValue{op: "+", val: value}
```

Comment:
`b.Set(column, Expr("? + ?", I(column), value)`
I did not test this, but you won't need to add opValue. you can also do the same for decrby.

Reply:
It works. Do we still need `IncrBy` and `DecrBy`? I think there is few people knows to use `Expr("? + ?", I(column), value)`. This two methods are more obvious to handle increasing and decreasing.

Reply:
This lgtm. Could you change the comment so that golint won’t complain . The method name is different from the one in comments